plugins/modules/content_view_filter.py

```diff
@@ -168,7 +168,7 @@
 
 RETURN = ''' # '''
 
-from ansible_collections.theforeman.foreman.plugins.module_utils.foreman_helper import KatelloAnsibleModule, _foreman_spec_helper
+from ansible_collections.theforeman.foreman.plugins.module_utils.foreman_helper import KatelloMixin, ForemanStatelessEntityAnsibleModule
```

Comment:
Should we have a `KatelloStatelessEntityAnsibleModule`?

Reply:
Maybe if we use it more than once?

Reply:
Yeah, I somehow read the diff as if we'd do that. But the other module is `setting`, so no.